src/main/java/com/sda/zdjavapol107/statki/ShipsMain.java

```diff
@@ -0,0 +1,283 @@
+                    beginOfTheGame();
+                    break;
+                }
+                case "2": {
+                    //TODO wyswietlanie list gier z gazy danych
+                    //TODO jezeli lista gier z bazy danych będzie różna od 0 możliwość wyboru gry przez uzytkownika
+                    UserInterface.printSingleLine("Wczytywanie");
+                    beginOfTheGame();
+                    break;
+                }
+                case "3": {
+                    mainLoop = false;
+                    break;
+                }
+                default: {
+                    UserInterface.printSingleLine("Podano błedna wartosc");
+                    break;
+                }
+            }
+        } while (mainLoop == true);
+        UserInterface.printSingleLine("Koniec gry");
+    }
+
+    private static void initialization() {
+        UserInterface.haveYouPlayed();
+        choiceUser = scanner.next();
+        if (choiceUser.equals("y")) {
+            //TODO wyswietlanie listy uzytkownikow z bazy danych
+            //TODO jezeli lista uzytkownikow z bazy danych będzie różna od 0 możliwość wyboru gry przez uzytkownika
+        } else {
+            UserInterface.creatingNewUser();
+            creatingNewUser();
+            UserInterface.placingTheShips();
+            do {
+                do {
+                    UserInterface.lastShipsToPlace();
+                    choiceShipSize = scanner.next();
+                    if (choiceShipSize.equals("1")) {
+                        if (fourSizeShip != 0) {
+                            haveShip = true;
+                        } else {
+                            haveShip = false;
+                            UserInterface.dontHaveAnyMoreShipsOfThatSize();
+                        }
+                    } else if (choiceShipSize.equals("2")) {
+                        if (threeSizeShip != 0) {
+                            haveShip = true;
+                        } else {
+                            haveShip = false;
+                            UserInterface.dontHaveAnyMoreShipsOfThatSize();
+                        }
+                    } else if (choiceShipSize.equals("3")) {
+                        if (twoSizeShip != 0) {
+                            haveShip = true;
+                        } else {
+                            haveShip = false;
+                            UserInterface.dontHaveAnyMoreShipsOfThatSize();
+                        }
+                    } else if (choiceShipSize.equals("4")) {
+                        if (oneSizeShip != 0) {
+                            haveShip = true;
+                        } else {
+                            haveShip = false;
+                            UserInterface.dontHaveAnyMoreShipsOfThatSize();
+                        }
+                    }
+                    if (haveShip == true) {
+                        switch (choiceShipSize) {
+                            case "1": {
```

Comment:
tutaj mozecie mi to pewnie jakos wytlumaczyc bo nie kumam ale mam takie spostrzezenie :D 
a. na pewno da sie bez tych ifow :D 
b. jak macie taki powtarzalny warunek jak w tym ifie to zamknijcie go w funkcji np areAllShipsDeployed czy cos w tym stylu :)